src/scss/components/_breadcrumbs.scss

```diff
@@ -32,23 +27,12 @@ $breadcrumb-chevron-point-right : -90deg;
     color: $grey--darkest;
     font-weight: $breadcrumb-font-weight;
 
-    //positioning for rotated chevron in :after
```

Comment:
Just to get up to speed on this – doesn't fozzie have a dependency on `f-icons`and so the SVG should be there for this inlining?

Reply:
Not at the moment, this is the quickest way to get it building again

Reply:
Ah, ok – guessing there's a task to add this back in then off the back of this?